### DISCUSSION TITLE
FATAL: Required dependency "node" is using an unsupported version.

### DISCUSSION THREAD

**Discussion by Crazygiscool** [BlueprintFramework/framework#64] [Support] [answered]
### Current Behavior

when I try to execute sudo bash blueprint.sh it give error FATAL: Required dependency "node" is using an unsupported version.

### Expected Behavior

install the framework

### Steps to Reproduce

in /var/www/pterodactly unziped release.zip and replaced everything, installed node and yarn with sudo, and sudo bash blueprint.sh

### Blueprint Version

beta-2024-08

### Panel Version

1.11.9

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details and tested my reproduction steps.
- [X] I have checked to my best capabilities and believe this is a bug with the software, and not a configuration issue with my specific system.
- [X] I have made sure to check that this bug is the fault of Blueprint and is not caused by stock Pterodactyl.

**Comment by Crazygiscool** [comment]
eh

**Comment by prplwtf** [comment, marked as answer]
Blueprint requires Node.js v17 and later, switch to a newer version and try again.
